src/backoff.erl

```diff
@@ -52,16 +72,30 @@ fire(#backoff{current=Delay, value=Value, dest=Dest}) ->
 -spec get(backoff()) -> pos_integer().
 get(#backoff{current=Delay}) -> Delay.
 
+%% Toggles jitter feature.
+-spec jitter(backoff()) -> backoff().
+jitter(B=#backoff{jitter=false}) ->
+    {A1,A2,A3} = now(),
+    random:seed(A1, A2, A3),
+    B#backoff{jitter=true};
```

Comment:
We should use a `maybe_seed` function here. Something like:

```erlang
maybe_seed() ->
case get(random_seed) of
    undefined -> random:seed(erlang:now());
    {X,X,X} -> random:seed(erlang:now());
    _ -> ok
end.
```

To avoid asking for `now()` just for a seed every time.